### PR TITLE
Add documentation for Braze Swift SDK setup

### PR DESCRIPTION
**Pull request by jerielng**
# Pull Request/Issue Resolution

Closes #**SDK-3023**

#### Is this change associated with a Braze feature/product release?
- [x] Yes (**12/15/2022**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [X] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [X] Check that all links work.
- [X] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [X] Tag others as reviewers as necessary.
- [x] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @timothy-kim or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Currents<br>Internal Tools<br>Product Partnerships<br>SMS<br>Customer Lifecycle, Identity and Permissions</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Reporting<br>Intelligence<br>User Targeting<br>IAM<br>Channels<br>FIX</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging and Automation<br>Email (Composition and Infrastructure)</td>
  </tr>
</table>
</details>

### CODE REVIEW

**Comment by josh-mccrowell-braze** [comment]
@cla-bot check

**Comment by davidbielik** [comment]
@josh-mccrowell-braze can you help figure out how to get "Swift" in the side panel to be uppercase? and to move it above "iOS" please?

**Comment by KellieHawks** [comment]
@cla-bot check

**Comment by lowip** [comment]
Also forgot to mention, I think we can swap the `{% tab swift %} and `{% tab OBJECTIVE-C %}` to ensure that Swift is the first language displayed

**Review by hokstuff — APPROVED** [review]
Looks good, great start to the Swift SDK docs project! Mainly some smaller nits

File: _docs/_developer_guide/home.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ guide_featured_list:
   image: /assets/img/android.png
   link: /docs/developer_guide/platform_integration_guides/android/initial_sdk_setup/android_sdk_integration/
   nav_link: android
-- name: iOS
-  link: /docs/developer_guide/platform_integration_guides/ios/initial_sdk_setup/overview/
+- name: iOS (Swift)
+  link: /docs/developer_guide/platform_integration_guides/swift/initial_sdk_setup/overview/
   fa_icon: fab fa-apple
   nav_link: ios
```
Comment (jerielng) [inline _docs/_developer_guide/home.md:17]: @josh-mccrowell-braze  Between this one and the `iOS (Objective-C)` section, should the `nav_link` be `ios` for both or should they be different?

Comment (josh-mccrowell-braze) [reply _docs/_developer_guide/home.md:17]: Pinging @KellieHawks for this question. I noticed other links had overlapping `nav_link` (e.g., "unity"), but wasn't sure.

Comment (KellieHawks) [reply _docs/_developer_guide/home.md:17]: Use "ios_c" and "ios_swift" for the nav links.

**Comment by jerielng** [comment]
@braze-inc/sdks-ios 

@josh-mccrowell-braze has updated the naming to be `Swift` and then `iOS (legacy)`, `macOS (legacy)`, etc. Could you give a thumbs up if that works with you?

**Pull request merged**